<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>826</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>14</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="title">
       <string>调试信息</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="5" column="1">
        <widget class="QComboBox" name="comboBox_wait_2">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>3kg</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>10kg</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>20kg</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>50kg</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QPushButton" name="pushButton_dis_2">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>清除校准</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QPushButton" name="pushButton_con_2">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>零点校准</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>量程选择</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="2">
        <widget class="QPlainTextEdit" name="plainTextEdit_2"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>14</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="title">
       <string>串口参数</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="9" column="1">
        <widget class="QPushButton" name="pushButton_con">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>连接</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="styleSheet">
          <string notr="true">font: 700 9pt &quot;微软雅黑&quot;;</string>
         </property>
         <property name="text">
          <string>停止位:1</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="styleSheet">
          <string notr="true">font: 700 9pt &quot;微软雅黑&quot;;</string>
         </property>
         <property name="text">
          <string>数据位:8</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QPushButton" name="pushButton_dis">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>断开</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0" colspan="2">
        <widget class="QPlainTextEdit" name="plainTextEdit"/>
       </item>
       <item row="7" column="1">
        <widget class="QComboBox" name="comboBox_wait">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>串口号：</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>9</pointsize>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 700 9pt &quot;微软雅黑&quot;;</string>
         </property>
         <property name="text">
          <string>波特率:115200</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>串口操作：</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="label_5">
         <property name="styleSheet">
          <string notr="true">font: 700 9pt &quot;微软雅黑&quot;;</string>
         </property>
         <property name="text">
          <string>校验位:无</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0" colspan="3">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1,1,1,2,1,1">
      <item>
       <widget class="QPushButton" name="pushButton_poll_2">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>14</pointsize>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;微软雅黑&quot;;</string>
        </property>
        <property name="text">
         <string>蓝牙扫描</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_poll">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>14</pointsize>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;微软雅黑&quot;;</string>
        </property>
        <property name="text">
         <string>蓝牙绑定</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_clear">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>14</pointsize>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;微软雅黑&quot;;</string>
        </property>
        <property name="text">
         <string>断开蓝牙连接</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_12">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>14</pointsize>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;微软雅黑&quot;;</string>
        </property>
        <property name="text">
         <string>蓝牙调试信息:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_13">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>14</pointsize>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;微软雅黑&quot;;</string>
        </property>
        <property name="text">
         <string>足底重心位置：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1" rowspan="2" colspan="2">
     <widget class="QWidget" name="widget" native="true"/>
    </item>
   </layout>
   <zorder>groupBox_2</zorder>
   <zorder>groupBox_3</zorder>
   <zorder>widget</zorder>
   <zorder></zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
